arm-billing/2017-02-27-preview/swagger/billing.json

```diff
@@ -0,0 +1,308 @@
+        "parameters": [
+          {
+            "name": "invoiceName", 
+            "in": "path",
+            "description": "The name of an invoice resource.",
+            "required": true,
+            "type": "string"
+          },   
+          {                                 
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+                "$ref": "#/definitions/Invoice"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/invoices/latest": {
+      "get": {
+        "tags": [
+          "Invoices"
+        ],
+        "operationId": "Invoices_GetLatest",
+        "description": "Gets the most recent invoice. When getting a single invoice, the downloadUrl property is expanded automatically.",
+        "x-ms-examples": {
+					"InvoicesGetLatest": {
+    				"$ref": "../examples/InvoicesGetLatest.json" 
+        	}
+        },
+        "parameters": [
+          {                                 
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+                "$ref": "#/definitions/Invoice"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Invoice": {
+      "description": "An invoice resource can be used download a PDF version of an invoice.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/InvoiceProperties"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "DownloadUrl": {
+      "description": "Secure URL that can be used temporarily to download a PDF invoice. May be null if returned from a list operation.",
```

Comment:
Don't describe in the description of a model type when it can be null, move that to the very operation it applies to.
Descriptions down here should really just describe the model, like the first sentence 👍